apps/web/src/pages/Departments/components/DepartmentTable.tsx

```diff
@@ -47,31 +46,29 @@ export const DepartmentTable = ({
     departmentsQuery,
   );
   const columns = [
+    columnHelper.accessor((row) => getLocalizedName(row.name, intl), {
+      id: "name",
+      sortingFn: normalizedText,
+      header: intl.formatMessage(commonMessages.name),
+      cell: ({ row: { original: department } }) => (
+        <Link href={paths.departmentView(department.id)}>
+          {getLocalizedName(department.name, intl)}
+        </Link>
+      ),
+      meta: {
+        isRowTitle: true,
+      },
+    }),
     columnHelper.accessor("departmentNumber", {
       id: "departmentNumber",
       filterFn: "weakEquals",
       header: intl.formatMessage({
-        defaultMessage: "Department #",
-        id: "QOvS1b",
+        defaultMessage: "Number",
```

Comment:
Everywhere else in the forms, it is referred to as Department number. Should it not be kept the same for consistency?